ShellPkg/Library/UefiShellDebug1CommandsLib/SetVar.c

```diff
@@ -438,6 +440,11 @@ ShellCommandRunSetVar (
         Status = gRT->GetVariable ((CHAR16 *)VariableName, &Guid, &Attributes, &Size, Buffer);
         if (Status == EFI_BUFFER_TOO_SMALL) {
           Buffer = AllocateZeroPool (Size);
+          if (Buffer == NULL) {

```

Comment:
MU_CHANGE tags?

Reply:
On the other lines above too.

Reply:
The lines above have tags from the last batch of ShellPkg CodeQL fixes, I just revised the changes to include the ShellPrintHiiEx() calls